crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,234 @@
+function CrowdsourceHinter(runtime, element, data){
+    //switching units back to a previous question will make a second hinter start up.
+    //executeHinter is used to disable the hinter after switching units in an edX course.
+    var executeHinter = true;
+    $(".crowdsourcehinter_block", element).hide();
+
+    if(executeHinter){
+    var isShowingHintFeedback = false;
+    var voted = false;
+    $(".csh_HintsToUse", element).text("");
+
+    function stopScript(){
+    //This function is used to prevent a particular instance of the hinter from acting after
+    //switching between edX course's units. 
+        executeHinter = false;
+    }
+    Logger.listen('seq_next', null, stopScript);
+    Logger.listen('seq_prev', null, stopScript);
+    Logger.listen('seq_goto', null, stopScript);
+
+    //data about the problem obtained from Logger.listen('problem_graded') is passed on to the onStudentSubmission.
+    //directly passing data to onStudentSubmission does not appear to work
```

Comment:
This is not enough documentation to fix the issue. We have two choices here: 

1. Fix it. 
2. Document what didn't work, so we can fix it later

Include the broken code in a comment, and give more context. It's okay if we leave debugging for later, but a future developer needs enough information to pick up where you left off. 

That future developer may be me, or even you in a year.